src/Integrations/ActiveCampaign/SettingsActiveCampaign.php

```diff
@@ -278,6 +278,13 @@ function ($option) use ($formId) {
 				$beforeContent = \apply_filters($filterName, '') ?? '';
 			}
 
+			$sortingButton = Components::render('sorting');
+
+			$formViewDetailsIsEditableFilterName = Filters::getIntegrationFilterName(self::SETTINGS_TYPE_KEY, 'fieldsSettingsIsEditable');
+			if (\has_filter($formViewDetailsIsEditableFilterName)) {
+				$sortingButton = \__('This integration sorting and editing is disabled because of the active filter in your project!', 'eightshift-forms');
```

Comment:
```suggestion
				$sortingButton = \__('Sorting and editing fields for this integration is disabled because of an active filter in your project!', 'eightshift-forms');
```